src/gluonts/ev/interface.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import ChainMap, Tuple, Optional, Iterable, Union
+from dataclasses import dataclass
+
+import numpy as np
+import pandas as pd
+
+from gluonts.dataset import DataEntry
+from gluonts.dataset.split import TestData
+from gluonts.ext.naive_2 import naive_2
+from gluonts.time_feature.seasonality import get_seasonality
+from gluonts.model.forecast import Forecast
+from gluonts.ev import seasonal_error
+
+
+@dataclass
+class ForecastDict:
+    """
+    Wrapper around ``Forecast`` objects, that adds ``__getitem__`` and
```

Comment:
I think `Forecast` objects already support this.

Reply:
Ah right, then I just need to add the batch dimension